up42/catalog.py

```diff
@@ -414,8 +431,7 @@ def construct_order_parameters(
         # Handled on API level, don't manipulate in SDK, providers might accept geometries in the future.
         if aoi is not None:
             aoi = any_vector_to_fc(vector=aoi)
-            aoi = fc_to_query_geometry(fc=aoi, geometry_operation="intersects")
-            order_parameters["params"]["aoi"] = aoi  # type: ignore
+            order_parameters["featureCollection"] = aoi  # type: ignore
```

Comment:
are you sure about this?
CMIIW but it seems the aoi is inside something like `["featureCollection"]["features"][0]["geometry"] `if looking at https://docs.up42.com/developers/api#tag/Orders/operation/createV2OrderUnderWorkspace

Reply:
To my understanding they are now asking for a GeoJSON in the featureCollection element. The aoi should be a geoJSON object and the function any_vector_to_fc converts any type of supported Geometry to featureCollection (fc). If you see now we are not getting the aoi from the feature collection, as it is not used anymore in the endpoint. 

Do you think I missed something?

Reply:
I will rename this variable.

Reply:
Ah ok, i want aware of what `fc_to_query_geometry` does. Nvm then, thank you